subsys/bluetooth/host/l2cap.c

```diff
@@ -872,6 +872,11 @@ static void l2cap_chan_tx_process(struct k_work *work)
 		if (sent < 0) {
 			if (sent == -EAGAIN) {
 				ch->tx_buf = buf;
+			} else {
+				/* The ownership of the buffer is with the stack
+				 * since bt_l2cap_chan_send has returned success
```

Comment:
Please remove the comment, since this is self-explanatory by the fact that this is an internal processing function that takes net_buf references from an internal queue, and therefore those are owned by the stack and not the application.

Reply:
Ok, so the comment generated more confusion than clarity, so we discussed it and decided it was best to just remove it.

Reply:
Will do :)